src/main/scala/edu/ie3/simona/model/thermal/CylindricalThermalStorage.scala

```diff
@@ -187,8 +187,10 @@ object CylindricalThermalStorage {
       input: CylindricalStorageInput,
       initialStoredEnergy: Energy = DefaultQuantities.zeroKWh,
   ): CylindricalThermalStorage = {
-    val minEnergyThreshold: Energy =
-      CylindricalThermalStorage.volumeToEnergy(
+    val minEnergyThreshold: Energy = {
+      // Temporary fix until changes in PSDM are released, Some minimumEnergyThreshold would lead to non-plausible behaviour
+      zeroKWh
+      /*CylindricalThermalStorage.volumeToEnergy(
```

Comment:
Does it make sense to remove the commented-out code already?

Reply:
We can do so, we have it in the version controll anyway...